src/app/content/studyGuides/selectors.ts

```diff
@@ -0,0 +1,12 @@
+import { createSelector } from 'reselect';
+import * as parentSelectors from '../selectors';
+
+export const localState = createSelector(
+  parentSelectors.localState,
+  (parentState) => parentState.studyGuides
+);
+
+export const studyGuidesSummary = createSelector(
+  localState,
+  (state) => state.summary
+);
```

Comment:
should move the enabled selector into here, and out of the base content selectors, same with the state